lms/djangoapps/experiments/utils.py

```diff
@@ -304,6 +304,75 @@ def get_experiment_user_metadata_context(course, user):
     context['has_staff_access'] = has_staff_access
     context['forum_roles'] = forum_roles
     context['partition_groups'] = user_partitions
+
```

Comment:
can you add to the docstring at the top of this function and explain that the purpose of this is to make data available for Optimizely experiments

Reply:
Sure- the first 40 lines already existed for the purpose in the existing comment, but I'll add a note about the new user_metadata element being added for our Optimizely experiments- thanks!

Reply:
all of this is being used for optimizely experiments, not just the new lines

Reply:
I see! Thanks for the clarification- updated the comment accordingly but let me know if that could be clearer